### PR TITLE
Display up-to-date node storage size

### PR DESCRIPTION
**Pull request by jlwllmr**
Adds functionality to update node volume size values weekly. 

1. The script `fetchNodeSize.js` is the main element. It sends the request to get the data, then writes the data to `linea-node-size/data.json`. It retrieves data for the nodes specified in `/linea-node-size/config.json`, which can be easily updated/modified if we want to use other nodes as sources.
2. Two React components then handle the implementation in the MDX files. `NodeSize.jsx` displays the values, while `LastUpdated.jsx` provides a timestamp. Both use the `data.json` as their source.
3. A Github Actions workflow will run weekly (00:00 on Mondays, UTC) and create a PR with the changes to the `data.json` file. Once merged, the node values will be refreshed. `data.json` should be the only item that requires updates, as the components will always pull the most recent data from the JSON.

image

Currently, the components are not present in the Erigon guide, as we need to establish to what extent this differs from Geth in terms of storage requirements. The Besu and Linea Besu guides use the same figures, as these should be identical in practice.

### CODE REVIEW

File: linea-node-size/config.json
Comment (diana-borbe-consensys) [inline linea-node-size/config.json:1]: lgtm

**Comment by mapachurro** [comment]
Hmmm 
image
Looking into this

**Review by mapachurro — APPROVED** [review]
I love this thing!

Happy to help you tweak the GHA if they don't work right out of the gate. :pray:

**Comment by jlwllmr** [comment]
Attempting to fix the fact the linter won't run. It seems we are somehow in a situation where:
- eslint >8.57.0 is not supported, so we should transition to eslint ^9.0.0. But that requires several additional packages
- These additional packages cause dependency issues with other eslint-related packages, when installed
- We already switched to the modern config format in #708 
- So setting up our repo for the linter to work would mean reverting to the older config format, and then awaiting all the plugins to catch up with eslint ^9.0.0 and become compatible

Going to merge anyway, may fix in a separate PR.

**Pull request merged**